### ISSUE TITLE
[NRCE] support for NODON SIN-2-2-01

### ISSUE THREAD

**Issue by Holger-Will** [enocean-js/enocean-js#127]

**Comment by Holger-Will** [comment]
so it seems the nodon module does not support `f6-03-01` but only `f6-02-01`... I did switch to 03-01 at some point because you can encode more buttons with it. but it seems there are a number of actors that do not support that eep. I'm thinking about making the eep used configurable and `f6-02-01` the default.

for the time being, you can use this flow as an example of how to send a switch command to the nodon module.

```
[{"id":"a9af1f2c.e31bf","type":"tab","label":"Flow 4","disabled":false,"info":""},{"id":"8f8233ef.4c982","type":"function","z":"a9af1f2c.e31bf","name":"","func":"var btnNum = {on:0,off:1}\nvar channel = 1\n\nvar btnClick = [\n        {\n            \"meta\": {\n                \"eep\": \"f6-02-01\",\n                \"channel\": channel,\n                \"type\": \"data\",\n                \"status\": 48\n            },\n            \"data\": {\n                \"R1\": btnNum[msg.payload],\n                \"EB\": 1\n            }\n        },\n        {\n            \"meta\": {\n                \"eep\": \"f6-02-01\",\n                \"channel\": channel,\n                \"type\": \"data\",\n                \"status\": 32\n            },\n            \"data\": {\n                \"R1\": btnNum[msg.payload],\n                \"EB\": 0\n            }\n        }\n    ]\nreturn {payload: btnClick}","outputs":1,"noerr":0,"x":640,"y":220,"wires":[["97730563.b62998"]]},{"id":"97730563.b62998","type":"split","z":"a9af1f2c.e31bf","name":"","splt":"\\n","spltType":"str","arraySplt":1,"arraySpltType":"len","stream":false,"addname":"","x":800,"y":220,"wires":[["4c10067d.6d6218"]]},{"id":"4c10067d.6d6218","type":"enocean-out","z":"a9af1f2c.e31bf","serialport":"5ffd4f8d.f6fab","name":"","logOutput":true,"x":960,"y":220,"wires":[[]]},{"id":"defcf021.686dd","type":"inject","z":"a9af1f2c.e31bf","name":"","topic":"","payload":"on","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":430,"y":180,"wires":[["8f8233ef.4c982"]]},{"id":"6d82a9de.5e4c28","type":"inject","z":"a9af1f2c.e31bf","name":"","topic":"","payload":"off","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":430,"y":240,"wires":[["8f8233ef.4c982"]]},{"id":"5ffd4f8d.f6fab","type":"enocean-config-node","z":"","serialport":"/dev/ttyUSB0"}]
```

**Comment by Holger-Will** [comment]
for `d2-01-12` to work, #91 must be implemented... working on a patch...

**Comment by Holger-Will** [comment]
getting closer... #91 is fixed in my local build and is working great. `d2-01-00` eep description is also fixed (#99 )
sending status request works as expected, and I'm getting a response.

**Comment by Holger-Will** [comment]
so now I got everything else working... still some code cleanup to do until I can publish...

**Comment by es101** [comment]
Thank you Holger.
Now I can use the actors with the above flow.

Will there be a better handling to set the channel in the published version.

Regards

Erwin

**Comment by Holger-Will** [comment]
the patch will contain a feature that allows you to use the 4-rocker node with your device.
even better yet you can use `d2-01-12` for all kinds of stuff.  I will publish an example flow when I'm finished...

in any case: to make handling easier, you can always use a function node...

**Comment by Holger-Will** [comment]
published v0.6.0

* support for UTE teach in response for bidirectional teach in (experimatal)
* fix eep d2-01-*
* fix generating 4BS teach in  telegrams

**Comment by Holger-Will** [comment]
example flow
```
[{"id":"a5e956de.749b68","type":"tab","label":"Flow 2","disabled":false,"info":""},{"id":"f23f04f5.51e628","type":"enocean-in","z":"a5e956de.749b68","serialport":"5ffd4f8d.f6fab","name":"in","x":90,"y":100,"wires":[["ca588d44.5387d"]]},{"id":"ca588d44.5387d","type":"enocean-actor","z":"a5e956de.749b68","name":"Aktor 1","channel":"10","x":280,"y":120,"wires":[["8741c76e.8ffdd8"],["fbdde8aa.bc7da8"]]},{"id":"539b496c.88c8c8","type":"inject","z":"a5e956de.749b68","name":"LRN","topic":"","payload":"{\"type\":\"LRN\",\"duration\":30}","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":90,"y":160,"wires":[["ca588d44.5387d"]]},{"id":"8741c76e.8ffdd8","type":"debug","z":"a5e956de.749b68","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","x":570,"y":80,"wires":[]},{"id":"fbdde8aa.bc7da8","type":"enocean-out","z":"a5e956de.749b68","serialport":"5ffd4f8d.f6fab","name":"","logOutput":true,"x":580,"y":160,"wires":[[]]},{"id":"fdc10fd0.cb5ff","type":"inject","z":"a5e956de.749b68","name":"on","topic":"","payload":"{\"meta\":{\"eep\":\"d2-01-12\",\"channel\":10,\"type\":\"data\",\"data\":1},\"data\":{\"CMD\":{\"value\":1},\"I/O\":{\"value\":30},\"OV\":{\"value\":100}}}","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":330,"y":200,"wires":[["fbdde8aa.bc7da8"]]},{"id":"cb982957.07f248","type":"inject","z":"a5e956de.749b68","name":"off","topic":"","payload":"{\"meta\":{\"eep\":\"d2-01-12\",\"channel\":10,\"type\":\"data\",\"data\":1},\"data\":{\"CMD\":{\"value\":1},\"I/O\":{\"value\":0},\"OV\":{\"value\":0}}}","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":330,"y":240,"wires":[["fbdde8aa.bc7da8"]]},{"id":"8b144f59.4041c","type":"inject","z":"a5e956de.749b68","name":"request state","topic":"","payload":"{\"meta\":{\"eep\":\"d2-01-12\",\"channel\":10,\"type\":\"data\",\"data\":3},\"data\":{\"CMD\":{\"value\":3},\"I/O\":{\"value\":30}}}","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":300,"y":280,"wires":[["fbdde8aa.bc7da8"]]},{"id":"da4ebd30.77203","type":"inject","z":"a5e956de.749b68","name":"request external interface","topic":"","payload":"{\"meta\":{\"eep\":\"d2-01-12\",\"channel\":10,\"type\":\"data\",\"data\":3},\"data\":{\"CMD\":{\"value\":12},\"I/O\":{\"value\":30}}}","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":260,"y":320,"wires":[["fbdde8aa.bc7da8"]]},{"id":"1f9d94dc.02bb7b","type":"inject","z":"a5e956de.749b68","name":"delay off 2s","topic":"","payload":"{\"meta\":{\"eep\":\"d2-01-12\",\"channel\":10,\"type\":\"data\",\"data\":11},\"data\":{\"CMD\":{\"value\":11},\"I/O\":{\"value\":30},\"AOT\":{\"value\":20}}}","payloadType":"json","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":310,"y":360,"wires":[["fbdde8aa.bc7da8"]]},{"id":"5ffd4f8d.f6fab","type":"enocean-config-node","z":"","serialport":"/dev/ttyUSB0"}]
¸¸¸
